utility/src/main/java/org/oppia/util/parser/HtmlParser.kt

```diff
@@ -0,0 +1,37 @@
+package org.oppia.util.data;
+
+import android.content.Context
+import android.text.Html
+import android.text.Spannable
+import android.widget.TextView
+
+const val CUSTOM_TAG = "oppia-noninteractive-image"
+const val HTML_TAG = "img"
+const val CUSTOM_ATTRIBUTE = "filepath-with-value"
+const val HTML_ATTRIBUTE = "src"
+
+/** Html Parser for android TextView to parse Html tag. */
+class HtmlParser(
+  internal var context: Context,
```

Comment:
Please make this & the next values private--they don't need to be retrieved outside of this class.

Reply:
done.